datalake/src/main/java/com/sequenceiq/datalake/flow/datalake/upgrade/handler/DatalakeVmReplaceWaitHandler.java

```diff
@@ -32,7 +32,7 @@
     @Value("${sdx.vm.replace.sleeptime_sec:20}")
     private int sleepTimeInSec;
 
-    @Value("${sdx.vm.replace.duration_min:60}")
+    @Value("${sdx.vm.replace.duration_min:120}")
```

Comment:
Is this 120 based on something, or just the usual make it double? :)

Reply:
it is based on the case medium duty template contains 5 host groups and we do upgrade group by group :D